commands/random/coinflip.js

```diff
@@ -0,0 +1,22 @@
+const {
+  SlashCommandBuilder,
+  EmbedBuilder,
+  AttachmentBuilder,
+} = require("discord.js");
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName("coinflip")
+    .setDescription("Flips a coin!"),
+  async execute(interaction) {
+    const randomNum = Math.round(Math.random());
+    const headsOrTails = randomNum == 0 ? "Heads!" : "Tails!";
+    const fileName = randomNum == 0 ? "Heads.png" : "Tails.png";
+    const file = new AttachmentBuilder(`./images/${fileName}`);
```

Comment:
Don't have local images. Put their image addresses there. It takes up less space and still loads just as fast.